doc/source/getting_started/comparison/comparison_with_spreadsheets.rst

```diff
@@ -96,6 +106,248 @@ In pandas, you pass the URL or local path of the CSV file to :func:`~pandas.read
+
+Once parsed, spreadsheets display the dates in a default format, though `the format can be changed
+<https://support.microsoft.com/en-us/office/format-a-date-the-way-you-want-8e10019e-d5d8-47a1-ba95-db95123d273e>`_.
+In pandas, you'll generally want to keep dates as ``datetime`` objects while you're doing
+calculations with them. Outputting *parts* of dates (such as the year) is done through `date
+functions
+<https://support.microsoft.com/en-us/office/date-and-time-functions-reference-fd1b5961-c1ae-4677-be58-074152f97b81>`_
+in spreadsheets, and :ref:`datetime properties <10min_tut_09_timeseries.properties>` in pandas.
+
+Given ``date1`` and ``date2`` in columns ``A`` and ``B`` of a spreadsheet, you might have these
+formulas:
+
+.. list-table::
+    :header-rows: 1
+    :widths: auto
+
+    * - column
+      - formula
+    * - ``date1_year``
+      - ``=YEAR(A2)``
+    * - ``date2_month``
+      - ``=MONTH(B2)``
+    * - ``date1_next``
+      - ``=DATE(YEAR(A2),MONTH(A2)+1,1)``
+    * - ``months_between``
+      - ``=DATEDIF(A2,B2,"M")``
+
+The equivalent pandas operations are shown below.
+
+.. include:: includes/time_date.rst
+
+See :ref:`timeseries` for more details.
+
+
+Selection of columns
+~~~~~~~~~~~~~~~~~~~~
+
+In spreadsheets, you can select columns you want by:
+
+- `Hiding columns <https://support.microsoft.com/en-us/office/hide-or-show-rows-or-columns-659c2cad-802e-44ee-a614-dde8443579f8>`_
+- `Deleting columns <https://support.microsoft.com/en-us/office/insert-or-delete-rows-and-columns-6f40e6e4-85af-45e0-b39d-65dd504a3246>`_
+- `Referencing a range <https://support.microsoft.com/en-us/office/create-or-change-a-cell-reference-c7b8b95d-c594-4488-947e-c835903cebaa>`_ from one worksheet into another
+
+Since spreadsheet columns are typically `named in a header row
+<https://support.microsoft.com/en-us/office/turn-excel-table-headers-on-or-off-c91d1742-312c-4480-820f-cf4b534c8b3b>`_,
+renaming a column is simply a matter of changing the text in that first cell.
+
+.. include:: includes/column_selection.rst
+
+
+Sorting by values
+~~~~~~~~~~~~~~~~~
+
+Sorting in spreadsheets is accomplished via `the sort dialog <https://support.microsoft.com/en-us/office/sort-data-in-a-range-or-table-62d0b95d-2a90-4610-a6ae-2e545c4a4654>`_.
+
+.. image:: ../../_static/spreadsheets/sort.png
+   :alt: Screenshot of dialog from Excel showing sorting by the sex then total_bill columns
+   :align: center
+
+.. include:: includes/sorting.rst
+
+String processing
+-----------------
+
+Finding length of string
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+In spreadsheets, the number of characters in text can be found with the `LEN
+<https://support.microsoft.com/en-us/office/len-lenb-functions-29236f94-cedc-429d-affd-b5e33d2c67cb>`_
+function. This can be used with the `TRIM
+<https://support.microsoft.com/en-us/office/trim-function-410388fa-c5df-49c6-b16c-9e5630b479f9>`_
+function to remove extra whitespace.
+
+::
+
+   =LEN(TRIM(A2))
```

Comment:
Just FYI, Pygments doesn't have support for Excel formulas (https://github.com/pygments/pygments/issues/1664) and flake8-rst seems to interpret `code-block`s without a language specified as Python, so included the Excel formulas as literal blocks.

Reply:
sgtm